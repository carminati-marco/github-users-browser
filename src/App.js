import React from "react";
import "bootstrap/dist/css/bootstrap.min.css";
import styled from "styled-components";

import { Container, Row, Col } from "react-bootstrap";
import logo from "./logo.png";
import UserList from "./components/UserList";
import UserDetail from "./components/UserDetail/index";

const Header = styled.header`
  background-color: #282c34;
  min-height: 72px;
  line-height: 72px;

  display: flex;
  flex-direction: column;
  align-items: flex-start;
  justify-content: left;
  color: #fff;

  &&&& img {
    padding: 4px;
  }
`;

const StyledContainer = styled(Container)`
  padding: 10px;
`;

function App() {
  return (
    <div>
      <Header>
        <img src={logo} alt="Github" />
      </Header>
      <StyledContainer fluid>
        <Row>
          <Col xs lg="3">
            <UserList />
          </Col>
          <Col>
            <UserDetail />
          </Col>
        </Row>
      </StyledContainer>
    </div>
  );
}

export default App;
