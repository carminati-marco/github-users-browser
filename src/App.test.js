import React from "react";
import { Provider } from "react-redux";
import { shallow } from "enzyme";
import App from "./App";

import store from "./redux/store";

test("shallow without crashing", () => {
  const component = shallow(
    <Provider store={store}>
      <App />
    </Provider>
  );
  expect(component).toMatchSnapshot();
});
