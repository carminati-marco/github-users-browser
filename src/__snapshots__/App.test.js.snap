// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <header
    className="sc-bxivhb gtUrrD"
  >
    <img
      alt="Github"
      src="test-file-stub"
    />
  </header>
  <div
    className="sc-ifAKCX fIKiaj container-fluid"
  >
    <div
      className="row"
    >
      <div
        className="col-lg-3 col"
      >
        <div>
          <div
            className="mb-3 input-group"
          >
            <input
              aria-describedby="basic-addon1"
              aria-label="Search"
              className="sc-bdVaJa eEvTZI form-control"
              onChange={[Function]}
              placeholder="Search.."
            />
          </div>
          <div
            className="list-group"
            onKeyDown={[Function]}
          >
            <div
              className="list-group-item"
              data-rb-event-key="1455447"
              disabled={false}
              onClick={[Function]}
            >
              rparent
            </div>
            <div
              className="list-group-item"
              data-rb-event-key="1455367"
              disabled={false}
              onClick={[Function]}
            >
              gtnx
            </div>
            <div
              className="list-group-item"
              data-rb-event-key="20163150"
              disabled={false}
              onClick={[Function]}
            >
              FredericBallot
            </div>
          </div>
        </div>
      </div>
      <div
        className="col"
      />
    </div>
  </div>
</div>
`;
